test/api.js

```diff
@@ -19,6 +19,21 @@ describe('client-side api', function () {
 				.call(done);
 		});
 
+		it('should ensure that duplicate bundleName-messageName pairs are ignored', done => {
+			e.apis.extension.listenFor('testMessageName', 'testBundleName', done);
+			e.apis.extension.listenFor('testMessageName', 'testBundleName', done);
```

Comment:
Hm, I'm pretty sure this should actually throw an error. If it's not, then something is wrong.

Though, to be honest, the design of this API where it only accepts one listener is pretty silly. There's no reason for us to have this limitation, though me changing that would probably be considered a breaking change? Not sure, would have to think about that.